src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -53,19 +53,42 @@
 
     private final String username;
 
-    private PostgresqlConnectionConfiguration(String applicationName, @Nullable Duration connectTimeout, @Nullable String database, boolean forceBinary, String host,
-                                              @Nullable Map<String, String> options, String password, int port, @Nullable String schema, String username) {
+    private final boolean ssl;
+
+    private final String sslCert;
```

Comment:
Nit: SSL fields except for `sslMode` should be `@Nullable`.

Reply:
Other fields has @Nullable only on their getters and constructor arguments, should I mark them nullable as well?